packages/ember-application/lib/system/application.js

```diff
@@ -1003,6 +1004,7 @@ Application.reopenClass({
 
     registry.injection('view', 'renderer', 'renderer:-dom');
     registry.register('view:select', SelectView);
+    registry.register('view:outlet', OutletView);
```

Comment:
I have no strong feelings about this, but why is it different from all these other registrations that are not prefixed? They all seem equally subject to collisions.